Add detailed HW reqs (#538)
smartcontractkit/documentation: docs/Additional Information/faq.md

## What are the hardware requirements for a Chainlink node?

Your Chainlink node should be run on a server that has a public IP address.

### Minimum

To get started running a Chainlink node, you will need a machine with at least **2 cores** and **4 GB of RAM**. 

### Recommended 

The requirements for running a Chainlink node scale as the number of jobs your node services also scales.  For nodes with over 100 jobs, you will need at least **4 cores** and **8GB of RAM**.  

Resources:

> *  <a href="https://docs.chain.link/docs/running-a-chainlink-node/" target="_blank">Running a Chainlink Node</a>

Reviewer: ```suggestion
*  [Running a Chainlink Node](../running-a-chainlink-node/)
```

Reviewer: Unless markdown format isn't working, avoid using html

*  <a href="https://github.com/smartcontractkit/chainlink/wiki/Development-Setup-Guide" target="_blank">Development Setup Guide</a>

## Database Requirements

In addition to running a Chainlink node, you will also need a PostgreSQL database.  Please use a version >= 11, and be sure that your DB host provides access to logs.

> ### Minimum

Reviewer: ```suggestion
### Basic Requirements
```

The minimum requirements for the database are **2 cores**, **4GB of RAM**, and **100 GB of storage**.

### Recommended

Similar to the Chainlink node, requirements increase as you service more jobs.  For more than 100 jobs, your database server will need at least **4 cores**, **16 GB of RAM**, and **100 GB of storage**. 

If using AWS, use dedicated IOPS (not the burstable one).

> ## Ethereum Client

Reviewer: ```suggestion
## Connecting to Ethereum Client
```

Connectivity to an Ethereum client is also required for communication with the blockchain. If you decide to run your own Ethereum client, you will want to run that on a separate machine. Hardware requirements of Ethereum clients may change over time.

Resources:
*  <a href="https://ethereum.stackexchange.com/a/27369" target="_blank">StackExchange Answer</a>

> *  <a href="https://ethereum.stackexchange.com/questions/tagged/hardware" target="_blank">All Ethereum Hardware Questions</a>

Reviewer: ```suggestion
*  [All Ethereum Hardware Questions](https://ethereum.stackexchange.com/questions/tagged/hardware)
```

## What are the hardware requirements for a Chainlink node?

Your Chainlink node should be run on a server that has a public IP address.

### Basic Requirements

To get started running a Chainlink node, you will need a machine with at least **2 cores** and **4 GB of RAM**. 

### Recommended Settings

The requirements for running a Chainlink node scale as the number of jobs your node services also scales.  For nodes with over 100 jobs, you will need at least **4 cores** and **8GB of RAM**.  

Resources:
*  [Running a Chainlink Node](../running-a-chainlink-node/)
*  <a href="https://github.com/smartcontractkit/chainlink/wiki/Development-Setup-Guide" target="_blank">Development Setup Guide</a>

## Database Requirements

In addition to running a Chainlink node, you will also need a PostgreSQL database.  Please use a version >= 11, and be sure that your DB host provides access to logs.

### Basic Requirements

The minimum requirements for the database are **2 cores**, **4GB of RAM**, and **100 GB of storage**.

### Recommended Settings

> Similar to the Chainlink node, requirements increase as you service more jobs.  For more than 100 jobs, your database server will need at least **4 cores**, **16 GB of RAM**, and **100 GB of storage**. 
> 
> If using AWS, use dedicated IOPS (not the burstable one).

Reviewer 2: ```suggestion
Similar to the Chainlink node, requirements increase as you service more jobs.  For more than 100 jobs, your database server will need at least **4 cores**, **16 GB of RAM**, and **100 GB of storage**. 

If you run your node on AWS, use an instance type with dedicated core time. [Burstable Performance Instances](https://aws.amazon.com/ec2/instance-types/#Burstable_Performance_Instances) have a limited number of [CPU credits](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/burstable-credits-baseline-concepts.html), so you should not use them to run Chainlink nodes that require consistent performance.
```

Copied over from the ./running-a-chainlink-node.md page.
